examples/encoding/drm_integrations/create_drm_encoding_with_verimatrix.py

```diff
@@ -0,0 +1,389 @@
+import json
+import requests
+import datetime
+from bitmovin import Bitmovin, Encoding, S3Output, H264CodecConfiguration, \
+    AACCodecConfiguration, H264Profile, StreamInput, SelectionMode, Stream, EncodingOutput, ACLEntry, ACLPermission, \
+    FMP4Muxing, MuxingStream, DashManifest, DRMFMP4Representation, FMP4RepresentationType, Period, \
+    VideoAdaptationSet, AudioAdaptationSet, ContentProtection, HTTPSInput, HlsManifest, VariantStream, \
+    AudioMedia, FairPlayDRM, TSMuxing, AWSCloudRegion
+from bitmovin import CENCDRM as CENCDRMResource
+from bitmovin.resources.models import CENCPlayReadyEntry, CENCWidevineEntry
+from bitmovin.errors import BitmovinError
+
+def main():
+
+    API_KEY = '<INSERT_YOUR_API_KEY>'
+
+    # https://<INSERT_YOUR_HTTP_HOST>/<INSERT_YOUR_HTTP_PATH>
+    HTTPS_INPUT_HOST = '<INSERT_YOUR_HTTPS_HOST>'
+    HTTPS_INPUT_PATH = '<INSERT_YOUR_HTTPS_PATH>'
+
+    S3_OUTPUT_ACCESSKEY = '<INSERT_YOUR_ACCESS_KEY>'
+    S3_OUTPUT_SECRETKEY = '<INSERT_YOUR_SECRET_KEY>'
+    S3_OUTPUT_BUCKETNAME = '<INSERT_YOUR_BUCKET_NAME>'
+
+    date_component = str(datetime.datetime.now()).replace(' ', '_').replace(':', '-').split('.')[0].replace('_', '__')
+    OUTPUT_BASE_PATH = '/encoding/{}/'.format(date_component)
+
+    contentId = "<YOUR_VERIMATRIX_CONTENT_ID>"
+    siteId = "<YOUR_VERIMATRIX_SITE_ID>"
+
+    contentURL = 'http://your.verimatrix.endpoint:8058/cei/v1.0/content/{}?siteId={}'
+    contentURL = contentURL.format(contentId, siteId)
+
+    contentRequest = """{
+        "contentRequest": {
+            "siteId": "",
+            "contentId": "",
+            "contentType": "VOD",
+            "streamingProtocol": "SS",
```

Comment:
streamingProtocol should probably be DASH or HLS in our case? Check the Verimatrix documentation or with them directly.